### DISCUSSION TITLE
[A-L] (2023/24) Foglio 16 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#291] [Esercizi - Viaggi/Piazza]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/577dfb54-fd90-4c54-84aa-50c60d57e756)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/347efa51-95ed-41c5-a43a-9ae8b8eb9769)

**Comment by Elia-Belli** [comment]
Lo sviluppo di Laplace più conveniente è quello per la prima riga:

$$det(A)=(-1)^2\cdot 1\cdot det(A_{11})=(-1)^4\cdot 2\cdot det(B_{31})+(-1)^6\cdot (-1)\cdot det(B_{33})$$

> Per comodità ho rinominato $A_{11}= B$

$$det(A)=2\cdot [k\cdot 1- (-1)\cdot (-1)]-[1\cdot (-1)-1\cdot k]=3k-1$$

Affinchè la matrice sia invertibile $det(A)\neq 0$:

$$3k-1\neq 0\to k\neq \frac{1}{3}$$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/ee7a71c5-f3aa-4d94-b1b7-a1e131ff0f02)
